src/lib.rs

```diff
@@ -922,13 +922,23 @@ impl MasteringMetadata {
     }
 }
 
+/// Store content encoding information
+#[derive(Clone, Debug)]
+pub enum ContentEncodingValue {
+    /// Other encoding not handled.
+    Unknown,
+    /// Indicate than content is compressed.
+    Compression(ContentCompression),
+    /// Indicate than content is encrypted. And add information
```

Comment:
I think you can remove "And add information" here.

Reply:
:ok: